inventory

```diff
@@ -17,7 +17,7 @@ openshift_install_examples=false
 openshift_master_default_subdomain=cloudapps.example.com
 
 # Image used for the kubevirt/web-ui deployment
-kubevirt_web_ui_image_name=docker.io/mareklibra/kubevirt-web-ui:v1.3-8
+kubevirt_web_ui_image_name=quay.io/kubevirt/kubevirt-web-ui:latest
```

Comment:
Can you please change it to a static version?
We would like to gate version changes using CI.

Reply:
sure, done.